benchmark/benchmark.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# CPU Solvers
+for i in `ls case*` 
+# for i in case300.m 
```

Comment:
maybe remove dead comment?

Reply:
> Nice pull request! Overall, I just have a few comments.
> I would suggest moving all instance data in the same folder. Currently, we have some redundancy between:
> `test/data`
> `benchmark`
> 
> I would move all instances in the same folder. Maybe in `test/data`, or in a new `data` folder. What do you think?

Yes. I think a data folder makes sense. Which though raises the question again of making the parser more general as a separate package. But that's for some other time I guess.

Reply:
This fixed by 48bab95. I still kept symbolic links in the benchmarks folder since not every case is benchmarked. Does that make sense @frapac ?